public/js/rules.js

```diff
@@ -0,0 +1,84 @@
+var Rule = React.createClass({
```

Comment:
We should stick with a consistent naming convention, i.e. make the first letter lowercase.